packages/edit-post/src/components/layout/index.js

```diff
@@ -217,7 +217,7 @@ function Layout( { styles } ) {
 						}
 					/>
 				}
-				editorNotices={ <EditorNotices /> }
+				editorNotices={ ! isDistractionFree && <EditorNotices /> }
```

Comment:
There's no need for a condition; it was intentional - https://github.com/WordPress/gutenberg/pull/41740/files#diff-8933c0b377d83066f7dde9c8b573bab9a7bfe60f73919e4e5e80aff92a88800cR224.

Reply:
Okay, I reverted 👍